php/5.3/Dockerfile

```diff
@@ -26,6 +26,8 @@ RUN echo "Starting ..." && \
     MAKEFLAGS=' -j8' LDFLAGS=-lstdc++ phpenv install ${PHP_VERSION} && \
     phpenv global ${PHP_VERSION} && \
     phpenv rehash && \
+    curl -sSL https://pear.php.net/go-pear.phar -o /tmp/go-pear.phar && \
+    php /tmp/go-pear.phar && \
```

Comment:
Can you update the `CHANGELOG.md` please?

Reply:
Sure, it is done and pushed